src/Controls/src/Core/WebView/WebView.cs

```diff
@@ -191,6 +191,11 @@ public void Reload()
 		/// </summary>
 		public event EventHandler<WebNavigatingEventArgs> Navigating;
 
+		/// <summary>
+		///  Raised when a WebView process ends unexpectedly.
+		/// </summary>
+		public event EventHandler ProcessTerminated;
```

Comment:
Is there any way for people to get to the platform-specific event args for this? They can contain useful info.

In Blazor what we've done (but not for this specific event) is we do like `EventHandler<WebViewProcessTerminatedEventArgs>` and then we have platform-specifics like this:

```c#
public class WebViewProcessTerminatedEventArgs : EventArgs
{
#if WINDOWS
    public WebView2Thingy WebView2Thingy { get; }
#elif ANDROID
    public AndroidThingy AndroidThingy { get; }
#endif
   // and maybe iOS/MacCat don't have any extra info at all
}
```